README-development.md

```diff
@@ -1,3 +1,39 @@
+# Operating System
+
+## Prerequisites
+
+- git
+- coreos-assembler
+- rpmdistro-gitoverlay
+- ostree
+- rpm-ostree 
+- imagefactory-plugins-TinMan
+- qemu-img
+
+
+## Building
+
+- Clone ``openshift/os``
+- Move into the cloned repo
+- Build packages from source repos:
+ - ``rpmdistro-gitoverlay init``
+ - ``rpmdistro-gitoverlay resolve --fetch-all``
+ - ``rpmdistro-gitoverlay build``
+- Make the ostree
+ - ``export REPO=/tmp/repo``
+ - ``ostree init --repo=$REPO --mode=archive --src $REPO/ --dest=$REPO``
+ - ``coreos-assembler --repo=$REPO host.yaml``
+ - ``ostree summary --repo=$REPO --update --dest $REPO/ --src=$REPO``
+- Make the qcow2
+ - ``ostree --repo=repo remote add rhcos --no-gpg-verify ${OSTREE_INSTALL_URL}``
+ - ``ostree --repo=repo pull --mirror --commit-metadata-only rhcos``
+ - ``imagefactory --debug base_image --file-parameter install_script cloud.ks --parameter offline_icicle True rhcos.tdl``
+ - ``export IMAGE=`ls /var/lib/imagefactory/storage/*.body```
```

Comment:
Oh neat! I didn't know that you could use double ticks on the outside to safely use single ticks `` ` `` inside them!

Reply:
I'm not sure you can :laughing: